a10_octavia/controller/worker/controller_worker.py

```diff
@@ -866,6 +876,7 @@ def create_l7rule(self, l7rule_id):
         :returns: None
         :raises NoResultFound: Unable to find the object
         """
+        time.sleep(60)
```

Comment:
Don't use sleep(60) but use a general API to check/re-check and wait instead.

Reply:
Fixed